src/main/java/com/meetup/server/user/dto/response/UserEventHistoryResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.meetup.server.user.dto.response;
+
+import com.meetup.server.event.domain.Event;
+import com.meetup.server.global.util.TimeUtil;
+import com.meetup.server.recommend.domain.RecommendPlace;
+import com.meetup.server.user.domain.User;
+import lombok.Builder;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Builder
+public record UserEventHistoryResponse(
+        UUID eventId,
+        String middlePointName,
+        Optional<String> placeName,
```

Comment:
```suggestion
        String placeName,
```
Optional은 데이터 직렬화를 할 수 없기 때문에 Optional를 필드로 사용하는 것은 권장하지 않아요!